readme.md

```diff
@@ -130,6 +130,15 @@ This output type gives the most information, but if verified files are long, it
 7 Seventh line
 ```
 
+Lastly, there is `OutputType.Minimal` which will show only the changed lines. Here is how the `Full` output looks like in `Minimal` mode.
```

Comment:
Given that `Full` is a valid value of the `OutputType` enum, I'd reword it like that to reduce possible confusion:

> Here is how the complete output looks like in `Minimal` mode.

Reply:
(or **entire** instead of **complete**, can't decide which one is better)